### DISCUSSION TITLE
leetcode-js/leetcode/ds/array

### DISCUSSION THREAD

**Discussion by giscus** [2xiao/leetcode-js#9] [Announcements]
# leetcode-js/leetcode/ds/array

2.1 数组 数组的定义 定义 数组（Array） 是一种线性表数据结构。它用一组连续的内存空间，来存储一组具有相同类型的数据。

https://2xiao.github.io/leetcode-js/leetcode/ds/array.html

**Comment by oorange-ocean** [comment]
感谢！写得很系统，对准备秋招的应届前端帮助很大~（顺便捉个虫，是Easy不是Esay

**Reply by 2xiao** [reply]
谢谢指正
